### ISSUE TITLE
Offer RDP/VNC as alternative to Input Bridge

### ISSUE THREAD

**Issue by pachuco** [olegos2/mobox#251]
In Mobox installer, can offer preconfigured rdp/vnc localhost connection option alongside Input Bridge.
This is worse for games, but much better for generic GUI and CLI apps., in particular for zooming and panning.

The user can bring his favorite control scheme and client(in my case, it's aRDP/bVNC with "simulated touchpad" input mode).

**Comment by Ma445-cell** [comment]
But dri3 won't work then

**Comment by pachuco** [comment]
Yes, it's a disadvantage for games. This scheme is meant for average GUI apps.
